build.sbt

```diff
@@ -1,15 +1,21 @@
 import scala.sys.process._
 
+import org.apache.commons.lang3.StringUtils
 import sbt.Tests._
 import sbt.Keys._
 import sbt.nio.Keys._
 
-val sparkVersionMajorMinor = "2.4"
-val sparkVersion = "2.4.3"
-
 lazy val scala212 = "2.12.8"
 lazy val scala211 = "2.11.12"
 
+lazy val sparkVersion = sys.env.getOrElse("SPARK_VERSION", "2.4.3")
+
+lazy val minorVersion = (version: String) =>
```

Comment:
nit: this should still be called `majorMinor`. Also, why is it a lazy val pointing to an anonymous function? Can it just be a lazy val, or at least a def?